styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -205,6 +216,64 @@ private void evaluateResourcesForDequeue(
         .map(resource -> ResourceWithInstance.create(resource, instanceState));
   }
 
+  @VisibleForTesting
+  void triggerBackfills(Collection<InstanceState> activeStates) {
+    final List<Backfill> backfills;
+    try {
+      backfills = storage.backfills();
+    } catch (IOException e) {
+      LOG.warn("Failed to get resource limits", e);
+      return;
+    }
+
+    final Map<String, Long> backfillStates = activeStates.stream()
+        // TODO: find out if backfill in other way that string comparison
+        .filter(state -> state.runState().data().triggerId().orElse("").startsWith("backfill-"))
+        .collect(groupingBy(
+            state -> state.runState().data().triggerId().orElse("UNKNOWN"),
+            HashMap::new,
+            counting()));
+
+    backfills.forEach(backfill -> {
+      final Optional<Workflow> workflowOpt = workflowCache.workflow(backfill.workflowId());
+
+      if (!workflowOpt.isPresent()) {
+        LOG.warn("workflow not found for backfill: {}", backfill);
```

Comment:
This doesn't seem like something that will be disappear in the next tick. Should we advance `nextTrigger` in this case? In the future we can store more information about the status of each trigger to explicitly point out to users that the issue with this was missed information from the schedule source.